package.json

```diff
@@ -36,9 +38,11 @@
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
     "eslint-plugin-vue": "^5.2.3",
+    "fast-xml-parser": "^3.15.1",
```

Comment:
see comment at https://github.com/alpheios-project/fixtures/commit/cb68c7a66dd3d9d8bddfffc1bd58571e1f81289c#r36521823 - there isn't any need to parse the xml here, we should use JSON in the fixtures.

Reply:
>I think the changes here are okay, but see my note on the fixtures repo. I really think we should just use the JSON from the service in our fixtures.

I am sorry! I missed this opportunity! Of course it is much better to simply use ready JSON-formatted files. Will change it!